CMakeLists.txt

```diff
@@ -84,6 +84,10 @@ option(use_http "set use_http to ON if http is to be used, set to OFF to not use
 option(use_mqtt "set use_mqtt to ON if mqtt is to be used, set to OFF to not use mqtt" ON)
 option(use_xplat_uuid "use the SDK's platform-independent UUID implementation (default is OFF)" OFF)
 
+if(WIN32)
+    option(use_etw "set use_etw to ON if ETW logging is to be used. Default is OFF" OFF)
```

Comment:
This part is weird. "If ETW logging is to be used" how? 😮 I'd really like to understand whether we **have** to add this to our root CMakeLists.txt file...I can't imagine we do. Instead, I suspect we can just pass `-Duse_etw=OFF` in the cmake command that builds whichever dependency that requires this.